test/test_field_reductions.jl

```diff
@@ -97,24 +97,24 @@ trilinear(x, y, z) = x + y + z
                 @test Txyz.operand.dims === (1, 2, 3)
                 @test wxyz.operand.dims === (1, 2, 3)
 
-                @test Txyz[1, 1, 1] ≈ 3
+                @test CUDA.@allowscalar Txyz[1, 1, 1] ≈ 3
                 @test Array(interior(Txy))[1, 1, :] ≈ [2.5, 3.5]
                 @test Array(interior(Tx))[1, :, :] ≈ [[2, 3] [3, 4]]
-                @test wxyz[1, 1, 1] ≈ 3
+                @test CUDA.@allowscalar wxyz[1, 1, 1] ≈ 3
                 @test Array(interior(wxy))[1, 1, :] ≈ [2, 3, 4]
                 @test Array(interior(wx))[1, :, :] ≈ [[1.5, 2.5] [2.5, 3.5] [3.5, 4.5]]
                 
                 @compute Txyz = Field(Average(T, condition=T.>3))
                 @compute Txy = Field(Average(T, dims=(1, 2), condition=T.>3))
                 @compute Tx = Field(Average(T, dims=1, condition=T.>2))
-                @test Txyz[1,1,1] ≈ 3.75
+                @test Txyz[1, 1, 1] ≈ 3.75
                 @test Array(interior(Txy))[1, 1, :] ≈ [3.5, 11.5/3]
                 @test Array(interior(Tx))[1, :, :] ≈ [[2.5, 3] [3, 4]]
 
                 @compute wxyz = Field(Average(w, condition=w.>3))
                 @compute wxy = Field(Average(w, dims=(1, 2), condition=w.>2))
                 @compute wx = Field(Average(w, dims=1, condition=w.>1))
```

Comment:
```suggestion
                @compute wxyz = Field(Average(w, condition=w>3))
                @compute wxy   = Field(Average(w, dims=(1, 2), condition=w>2))
                @compute wx     = Field(Average(w, dims=1, condition=w>1))
```

Reply:
then for this to work we have to add:
```
import Base: |, &, >, <, ==, ≈, <=, >=
@binary >
@binary <
@binary ==
@binary ≈
@binary <=
@binary >=

@binary |
@binary &
```
to AbstractOperations.jl

Reply:
sorry you also have to add this line to `conditional_operations.jl` 😅 
```
@inline condition_operand(func::Function, operand::AbstractField, condition::AbstractField, mask) = ConditionalOperation(operand; func, condition, mask)
```

Reply:
done

Reply:
a lot of other tests were breaking so perhaps we leave this for a future PR?